x-pack/plugin/core/src/main/resources/logs-default-pipeline.json

```diff
@@ -0,0 +1,24 @@
+{
+  "processors": [
+    {
+      "set": {
+        "description": "If '@timestamp' is missing, set it with the ingest timestamp",
+        "field": "@timestamp",
+        "override": false,
+        "copy_from": "_ingest.timestamp"
+      }
```

Comment:
@joegallo FYI this is how we're setting a default timestamp. We'll want to have millisecond precision so we're not truncating here. As the `@timestamp` field is mapped as a `date`, not `date_nanos`, we don't need to truncate microseconds and nanoseconds, I think.

Reply:
A `set` w/ `copy_from` is very fast. 👍 and ❤️